dapr/proto/runtime/v1/appcallback.proto

```diff
@@ -53,6 +53,13 @@ service AppCallbackHealthCheck {
   rpc HealthCheck(google.protobuf.Empty) returns (HealthCheckResponse) {}
 }
 
+// AppCallbackAlpha V1 is an optional extension to AppCallback V1 to opt
+// for Alpha RPCs.
+service AppCallbackAlpha {
```

Comment:
This is a common service added for all Alpha APIs. Once the rpc graduates to stable, AppCallback can be changed to include this rpc and rpc removed from this one.

Reply:
What would happen when we deprecate that service, however? Would apps that depend on the SDKs and implement `AppCallbackAlpha` crash unless they're updated?

Reply:
My understanding is that even if there is no rpc in this service, we will simply keep it there, so that any new Alpha rpc can be added to it (ref: https://github.com/dapr/dapr/issues/2218#issuecomment-1258499437). 
Now, when a rpc gets out of this service, it may either become stable and go to `AppCallback`, as a breaking change; or it may be deprecated as a whole.